test/functorch/test_aotdispatch.py

```diff
@@ -310,7 +317,7 @@ def f(a):
 
     def test_input_mutation_multiple(self):
         def f(a, b, c):
-            b.add_(1)
+            a.add_(1)
             c.add_(1)
```

Comment:
Yeah, your problem is you're testing input mutation with addition. d/dx (x + 1) = 1; you get the same gradient no matter what. Change these to `mul_(2)` and you should start seeing numeric failures.

Reply:
Ah yep - thanks. FWIW as of my changes last night, I think I now handle this case correctly (just changed to `.mul_(2)` locally and the tests still pass). The PR still isn't ready for review though, I'll give a shout when it is.